### PR TITLE
Plans: Update live course offering to Business onboarding session

### PR DESCRIPTION
**Pull request by jeremeylduvall**
Instead of regular live courses, we're now offering 1-on-1 onboarding sessions
for new Business customers (context: p7DVsv-1wD-p2). This updates the instances of "live courses" throughout the plan features pages and after checkout to now offer these 1-on-1 sessions instead.

Here's the wording you should expect to see:

Title: Get Personalized Help
Description: Schedule a one-on-one orientation with a Happiness Engineer to set up your site and learn more about WordPress.com.
CTA: Schedule a session
Link: https://calendly.com/wordpressdotcom/wordpress-com-business-site-setup/

To test, load up this branch and take a look at the following three areas:

http://calypso.localhost:3000/plans/

screen shot 2017-01-13 at 4 49 10 pm

http://calypso.localhost:3000/plans/my-plan/

screen shot 2017-01-13 at 4 49 40 pm

http://calypso.localhost:3000/checkout/thank-you/site.wordpress.com/ (Purchase a Business plan on your site)

receipt

I also did some minor housekeeping updating capitalization for consistency across plan offerings.

- http://calypso.localhost:3000/plans/ (All words capitalized)
- http://calypso.localhost:3000/plans/my-plan/ (Only first word capitalized)

### CODE REVIEW

File: client/blocks/product-purchase-features/product-purchase-features-list/business-onboarding.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import analytics from 'lib/analytics';
+import PurchaseDetail from 'components/purchase-detail';
+
+function trackOnboardingButtonClick() {
+	analytics.tracks.recordEvent( 'calypso_plan_features_onboarding_click' );
```
Comment (gwwar) [inline client/blocks/product-purchase-features/product-purchase-features-list/business-onboarding.jsx]: Optional, but we can avoid a direct analytics lib call ( this makes testing components easier) by passing through a click event:

in business-onboarding.jsx
```javascript
import { noop } from 'lodash'; 
export default localize( ( { translate, onClick = noop } ) => {
    //...
     <PurchaseDetail onClick={ onClick }
   //... 
}
```

In the parent component:
```javascript
import { partial } from 'lodash';

import { recordTracksEvent } from 'state/analytics/actions';
//...

render() {
    //...
    			<BusinessOnboarding
				key="businessOnboarding"
                                onClick={ recordBusinessOnboardingClick }
			/>,
    //...
}

//... 
export default connect(
	( state ) => {
		//...
	},
        {
             recordBusinessOnboardingClick: partial( recordTracksEvent, 'calypso_plan_features_onboarding_click' )
        }
)( ProductPurchaseFeaturesList );
```

**Comment by jeremeylduvall** [comment]
Thanks @gwwar! Updated with your suggestion. Still works as advertised. Will ship this today.

File: client/blocks/product-purchase-features/product-purchase-features-list/advertising-removed.jsx [hunk 2]
```diff
@@ -14,7 +14,7 @@ export default localize( ( { isBusinessPlan, translate } ) => {
 		<div className="product-purchase-features-list__item">
 			<PurchaseDetail
 				icon="speaker"
-				title={ translate( 'Advertising Removed' ) }
+				title={ translate( 'Advertising removed' ) }
```
Comment (a8ci18n) [inline client/blocks/product-purchase-features/product-purchase-features-list/advertising-removed.jsx]: Warning: 21 translations will be lost with this change.
Alternate string suggestion: 
* `translate( 'Remove all WordPress.com advertising from your website' )`&mdash; translations: **19**. 

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/remove-live-course-advertising)

File: client/my-sites/upgrades/checkout-thank-you/business-plan-details.jsx [hunk 3]
```diff
@@ -30,13 +29,13 @@ const BusinessPlanDetails = ( { selectedSite, sitePlans, selectedFeature } ) =>
...
-				description={ i18n.translate( 'Register for one of our live courses led by Happiness Engineers ' +
-					'to get the most out of your site.'
+				title={ i18n.translate( 'Get personalized help' ) }
+				description={ i18n.translate( 'Schedule a one-on-one orientation with a Happiness Engineer to set up ' +
+					'your site and learn more about WordPress.com.'
 				) }
-				buttonText={ i18n.translate( 'Register for a course' ) }
-				href={ support.CALYPSO_COURSES }
-				onClick={ trackCoursesButtonClick } />
+				buttonText={ i18n.translate( 'Schedule a session' ) }
```
Comment (a8ci18n) [inline client/my-sites/upgrades/checkout-thank-you/business-plan-details.jsx]: Alternate string suggestion: 
* `translate( 'Schedule a Session' )`&mdash; translations: **0**. 

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/remove-live-course-advertising)

File: client/my-sites/upgrades/checkout-thank-you/business-plan-details.jsx [hunk 4]
```diff
@@ -30,13 +29,13 @@ const BusinessPlanDetails = ( { selectedSite, sitePlans, selectedFeature } ) =>
 
 			<PurchaseDetail
 				icon="help"
-				title={ i18n.translate( 'Attend a live course' ) }
-				description={ i18n.translate( 'Register for one of our live courses led by Happiness Engineers ' +
-					'to get the most out of your site.'
+				title={ i18n.translate( 'Get personalized help' ) }
```
Comment (a8ci18n) [inline client/my-sites/upgrades/checkout-thank-you/business-plan-details.jsx]: Alternate string suggestion: 
* `translate( 'Personalize' )`&mdash; translations: **45**. 

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/remove-live-course-advertising)

File: client/blocks/product-purchase-features/product-purchase-features-list/monetize-site.jsx [hunk 5]
```diff
@@ -21,7 +21,7 @@ export default localize( ( { selectedSite, translate } ) => {
 						'WordAds lets you earn money by displaying promotional content.'
 					)
 				}
-				buttonText={ translate( 'Start Earning' ) }
+				buttonText={ translate( 'Start earning' ) }
```
Comment (a8ci18n) [inline client/blocks/product-purchase-features/product-purchase-features-list/monetize-site.jsx]: Warning: 19 translations will be lost with this change.
Alternate string suggestion: 
* `translate( 'Earnings' )`&mdash; translations: **21**. 

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/remove-live-course-advertising)

File: client/lib/plans/constants.js [hunk 6]
```diff
@@ -512,7 +512,7 @@ export const FEATURES_LIST = {
 
 	[ FEATURE_UPLOAD_PLUGINS ]: {
 		getSlug: () => FEATURE_UPLOAD_PLUGINS,
-		getTitle: () => i18n.translate( 'Install plugins' ),
+		getTitle: () => i18n.translate( 'Install Plugins' ),
```
Comment (a8ci18n) [inline client/lib/plans/constants.js]: Warning: 22 translations will be lost with this change.
Alternate string suggestion: 
* `translate( 'Installed Plugins' )`&mdash; translations: **77**. 

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/remove-live-course-advertising)

File: client/lib/plans/constants.js [hunk 7]
```diff
@@ -494,11 +494,11 @@ export const FEATURES_LIST = {
 		getStoreSlug: () => 'no-adverts/no-adverts.php'
 	},
 
-	[ FEATURE_LIVE_COURSES ]: {
-		getSlug: () => FEATURE_LIVE_COURSES,
-		getTitle: () => i18n.translate( 'Attend live courses' ),
+	[ FEATURE_BUSINESS_ONBOARDING ]: {
+		getSlug: () => FEATURE_BUSINESS_ONBOARDING,
+		getTitle: () => i18n.translate( 'Get Personalized Help' ),
```
Comment (a8ci18n) [inline client/lib/plans/constants.js]: Alternate string suggestion: 
* `translate( 'Personalize' )`&mdash; translations: **45**. 

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/remove-live-course-advertising)

File: client/my-sites/upgrades/checkout-thank-you/custom-domain-purchase-detail.jsx [hunk 8]
```diff
@@ -14,7 +14,7 @@ const CustomDomainPurchaseDetail = ( { selectedSite, hasDomainCredit, translate,
 	if ( hasDomainCredit && selectedSite.plan.user_is_owner && ! isPressableSite ) {
 		return ( <PurchaseDetail
 				icon="globe"
-				title={ translate( 'Select Your Custom Domain' ) }
+				title={ translate( 'Select your custom domain' ) }
```
Comment (a8ci18n) [inline client/my-sites/upgrades/checkout-thank-you/custom-domain-purchase-detail.jsx]: Warning: 21 translations will be lost with this change.
[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/remove-live-course-advertising)

File: client/my-sites/upgrades/checkout-thank-you/google-voucher/index.jsx [hunk 9]
```diff
@@ -112,7 +112,7 @@ class GoogleVoucherDetails extends Component {
 			<div className="google-voucher__initial-step">
 				<PurchaseButton
 					onClick={ this.onGenerateCode }
-					text={ this.props.translate( 'Generate Code' ) } />
+					text={ this.props.translate( 'Generate code' ) } />
```
Comment (a8ci18n) [inline client/my-sites/upgrades/checkout-thank-you/google-voucher/index.jsx]: Warning: 18 translations will be lost with this change.
Alternate string suggestion: 
* `translate( 'Generate Backup Codes' )`&mdash; translations: **30**. 

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/remove-live-course-advertising)

File: client/lib/plans/constants.js [hunk 10]
```diff
@@ -545,7 +545,7 @@ export const FEATURES_LIST = {
 
 	[ FEATURE_UPLOAD_PLUGINS ]: {
```
Comment (a8ci18n) [inline client/lib/plans/constants.js]: Alternate string suggestion: 
* `translate( 'Personalize' )`&mdash; translations: **45**. 

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/remove-live-course-advertising)

Comment (yoavf) [inline client/lib/plans/constants.js]: (down-voted because of wrong location)

**Pull request merged**